apps/web-giddh/src/app/app.module.ts

```
@@ -38,40 +38,44 @@ import { ScrollingModule } from '@angular/cdk/scrolling';
-// Application wide providers
+// Get white label configuration from localStorage
+const whiteLabelString = localStorage.getItem('whiteLabel');
+let whiteLabelConfig = whiteLabelString ? JSON.parse(whiteLabelString) : null;
 const APP_PROVIDERS = [
     ...APP_RESOLVER_PROVIDERS,
-    { provide: APP_BASE_HREF, useValue: IS_ELECTRON_WA ? './' : AppUrl + APP_FOLDER }
+    {
+        provide: APP_BASE_HREF,
+        useValue: IS_ELECTRON_WA
+            ? './'
+            : whiteLabelConfig && whiteLabelConfig.body && whiteLabelConfig.body.giddhWhiteLabel.domainName
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)